operations/multi-tenancy/ExampleWorkflow.md

```diff
@@ -0,0 +1,806 @@
+* Example with no tenants:
+    * For primary `user` namespace:
+        * `SwitchType=switch/hpe_slingshot`
+        * `SwitchParameters=vnis=1025-65535`
+* Example with one tenant:
+    * For primary `user` namespace:
+        * `SwitchType=switch/hpe_slingshot`
+        * `SwitchParameters=vnis=1025-32767`
+    * For `vcluster-noten01a-slurm` namespace:
+        * `SwitchType=switch/hpe_slingshot`
+        * `SwitchParameters=vnis=32768-65535`
+* Example with two tenants:
+    * For primary `user` namespace:
+        * `SwitchType=switch/hpe_slingshot`
+        * `SwitchParameters=vnis=1025-32767`
+    * For `vcluster-noten01a-slurm` namespace:
+        * `SwitchType=switch/hpe_slingshot`
+        * `SwitchParameters=vnis=32868-57353`
+    * For `vcluster-noten02a-slurm` namespace:
+        * `SwitchType=switch/hpe_slingshot`
+        * `SwitchParameters=vnis=57354-65535`
+
+#### `Slurmcluster` Partitions and Nodes
+
+The general advice to tailor the compute node configuration for each tenant is to look at the `slurm.conf` for the primary (`user` namespace) Slurm instance.
+Borrow the `NodeSet`, `PartitionName`, and `NodeName` directives that apply to each tenant.
+
+* In this example, we have 'moved' two Compute nodes to the `slurm.conf` in namespace `vcluster-noten01a-slurm`.
+
+    ```ini
+    # PARTITIONS
+    NodeSet=Compute Feature=Compute
+    PartitionName=workq Nodes=Compute MaxTime=INFINITE State=UP OverSubscribe=EXCLUSIVE
+    # BEGIN COMPUTE NODES
+    NodeName=nid[001002-001003] Sockets=2 CoresPerSocket=64 ThreadsPerCore=2 RealMemory=456704 Feature=Compute
+    ```
+
+#### `Slurmcluster` Secrets (for `nonroot` users)
+
+These configuration settings are made in each `Slurmcluster's` `/etc/sssd/sssd.conf` file (in each `slurmctld` pod)
+You should not need to create or edit the `sssd.conf` file.
+Simply clone that file from the primary `Slurmcluster` (`user` namespace) to each tenant namespace.
+
+### USS Configuration
+
+These changes are made to the `uss-config-management` git repo.
+Tenants are disambiguated by their HSM group name (e.g. `hsmgrouplabel` `"noten01a"` in `noten01a.yaml`, and `hsmGroup` `"noten01a"` in `nocls01a.yaml`).
+All tenants can be booted and configured with a single CFS configuration that contains the appropriate git commit ID in the USS layers.
+
+* Example: `group_vars/noten01a/slurm.yml`
+
+    ```yaml
+    munge_vault_path: secret/slurm/vcluster-noten01a-slurm/nocls01a/munge
+    slurm_conf_url: https://rgw-vip.local/wlm/vcluster-noten05-slurm/nocls01a/
+    slurmd_options: "--conf-server 10.156.124.104,10.156.124.105"
+    ```
+
+* Example:  `group_vars/noten02a/slurm.yml`
+
+    ```yaml
+    munge_vault_path: secret/slurm/vcluster-noten02a-slurm/nocls02a/munge
+    slurm_conf_url: https://rgw-vip.local/wlm/vcluster-noten02a-slurm/nocls02a/
+    slurmd_options: "--conf-server 10.156.124.108,10.156.124.109"
+    ```
+
+## Step-by-Step Guide
```

Comment:
The following sections should use an ordered list instead of an unordered list since it is sequential steps.